agent/containers/images/Makefile

```diff
@@ -293,6 +294,15 @@ centos-7-base.Dockerfile: _PKGMGR = yum
 # Push local images for the given tag and distribution.
 #-
 
+# Special push target for the CI which pushes all the container images for the
+# distribution to the specified registry with the specified tag.
+$(_DEFAULT_DISTROS:%=%-push-ci): %-push-ci:
+	for image in base tools tool-meister tool-data-sink workloads all; do \
```

Comment:
I find the definition and context of `all` a bit murky; but seeing a list of individual targets *plus* `all` just looks weird without any explanation. 😁

Reply:
😁  But, `all` _is_ one of the "individual targets" -- if you want to build everything, then you have to use the `everything` target!  😆  (Seriously...I didn't design this thing, I'm just trying to extend it.)

FWIW, the `all` image (e.g., `pbench-agent-all-fedora-36`) contains everything which the previous five images contain...I think -- it contains "all the Agent things".

These tokens are used throughout the makefile to build the pattern targets, but they are only collected together into one place in the `push` script, and I copied the list from there.  (I wonder if there is some phrasing which avoids using the word "all" for the comment at line 297....)